arkime/scripts/initarkime.sh

```diff
@@ -100,6 +100,17 @@ if [[ "$MALCOLM_PROFILE" == "malcolm" ]]; then
 
   fi # if/else OpenSearch database initialized
 
+  if [[ -n $ELASTICSEARCH_ILM_ENABLED ]]; then
+    if [[ -n $ELASTICSEARCH_ILM_HOT_WARM_ENABLED ]]; then
+      # set up Elasticsearch ILM Policy
+      $ARKIME_DIR/db/db.pl $DB_SSL_FLAG "${OPENSEARCH_URL_FULL}" ilm "${ELASTICSEARCH_ILM_OPTIMIZATION_PERIOD}" "${ELASTICSEARCH_ILM_RETENTION_TIME}" --hotwarm --segments "${ELASTICSEARCH_ILM_SEGMENTS}" --replicas "${ELASTICSEARCH_ILM_OLDER_SESSION_REPLICAS}" --history "${ELASTICSEARCH_ILM_HISTORY_RETENTION_WEEKS}"
```

Comment:
If the only difference between these commands is the `--hotwarm` flag, you could avoid the duplication of the rest of it by doing something like this, getting rid of the `if` statement right above this and doing something like:

```bash
[[ "${ELASTICSEARCH_ILM_HOT_WARM_ENABLED:-false}" == "true" ]] && HOT_WARM_FLAG=--hotwarm || HOT_WARM_FLAG=
$ARKIME_DIR/db/db.pl $DB_SSL_FLAG "${OPENSEARCH_URL_FULL}" ilm "${ELASTICSEARCH_ILM_OPTIMIZATION_PERIOD}" "${ELASTICSEARCH_ILM_RETENTION_TIME}" ${HOT_WARM_FLAG} --segments "${ELASTICSEARCH_ILM_SEGMENTS}" --replicas "${ELASTICSEARCH_ILM_OLDER_SESSION_REPLICAS}" --history "${ELASTICSEARCH_ILM_HISTORY_RETENTION_WEEKS}"
```

That would replace lines 104 to 109 with these two lines.